sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/AccessControlHelloWorld.java

```diff
@@ -23,10 +23,10 @@ public class AccessControlHelloWorld {
      */
     public static void main(String[] args) {
         /* Instantiate a KeyVaultAccessControlClient that will be used to call the service. Notice that the client is
-        using default Azure credentials. To make default credentials work, ensure that environment variables
-        'AZURE_CLIENT_ID', 'AZURE_CLIENT_KEY' and 'AZURE_TENANT_ID' are set with the service principal credentials.
+        using default Azure credentials. To make default credentials work, ensure that the environment variable
```

Comment:
I wonder if it's even worth duplicating this documentation, or just link to the DefaultAzureCredential docs. I suppose it doesn't hurt here, but the code sample itself doesn't really change: just use a DefaultAzureCredential. Could use env vars, Azure CLI context, VS creds, etc.

Reply:
I linked to the `DefaultAzureCredential` docs.